Documentation/operations/upgrade.rst

```diff
@@ -339,6 +339,11 @@ Annotations:
   is configured in native routing mode and :ref:`DSR mode with Geneve` is enabled.
   Explicitly configure ``--tunnel-protocol=geneve`` (or the equivalent
   ``tunnelProtocol=geneve`` helm value) when DSR with Geneve is enabled.
+* ``CILIUM_PREPEND_IPTABLES_CHAIN`` environment variable has been deprecated and
+  cannot be used anymore to enable or disable Cilium iptables rules prepend instead
+  of append.
+  Please use the ``CILIUM_PREPEND_IPTABLES_CHAINS`` environment variable (note the
+  trailing ``S``) or the command line flag ``--prepend-iptables-chains`` instead.
```

Comment:
Can we change up the wording here to include a bit of information regarding why this change was made? Maybe something like:

```suggestion
* The ``CILIUM_PREPEND_IPTABLES_CHAIN`` environment variable has been renamed
  to ``CILIUM_PREPEND_IPTABLES_CHAINS`` (note the trailing ``S``) to more accurately
  match the name of the associated command line flag ``--prepend-iptables-chains``.
```

Reply:
Fixed, thanks!